sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_base_client.py

```diff
@@ -24,8 +24,8 @@ def __init__(self, endpoint, credential, **kwargs):
         # type: (str, Union[AzureKeyCredential, TokenCredential], Any) -> None
         self._endpoint = endpoint
         self._credential = credential
-        self.api_version = kwargs.pop('api_version', FormRecognizerApiVersion.V2_1_PREVIEW)
-        validate_api_version(self.api_version)
+        self._api_version = kwargs.pop('api_version', FormRecognizerApiVersion.V2_1_PREVIEW)
```

Comment:
nit: should we add this to the changelog. sorry this was not a ba

Reply:
😆 good point